sktime/forecasting/base/_fh.py

```diff
@@ -95,19 +107,20 @@ def _check_values(values: Union[VALID_FORECASTING_HORIZON_TYPES]) -> pd.Index:
     # isinstance() does not work here, because index types inherit from each
     # other,
     # hence we check for type equality here
-    if type(values) in VALID_INDEX_TYPES:
+    if is_in_valid_index_types(values) or is_integer_index(values):
```

Comment:
this should be `is_relative_fh_type or is_absolute_fh_type` imo, no?

Reply:
better probably `is_in_valid_index_types` once it´s changed (see below)

Reply:
done